pytest_metadata/plugin.py

```diff
@@ -27,6 +27,17 @@ def metadata(pytestconfig):
     return pytestconfig._metadata
 
 
+def pytest_addoption(parser):
+    group = parser.getgroup('selenium', 'selenium')
```

Comment:
This shouldn't be in the 'selenium' group. If you just add it directly to the parser without a group then it will appear in the 'custom options'. For example: https://github.com/pytest-dev/pytest-base-url/blob/e1ec0aa7570937eda5e40727986c1914bf1f99b1/pytest_base_url/plugin.py#L51-L55